app/lib/common/themes/app_theme.dart

```diff
@@ -39,257 +40,32 @@ class EmojiConfig {
   static final emojiSizeMax = 32 * ((!kIsWeb && Platform.isIOS) ? 1.30 : 1.0);
 }
 
-AppTheme currentTheme = AppTheme();
-
 extension CustomColorScheme on ColorScheme {
-  Color get success => const Color(0xFF67A24A);
-  Color get tertiary2 => const Color(0xFFFFC333);
+  Color get primary => const Color(0xFF9CCAFF);
+  Color get tertiary => const Color(0xFFFFC333);
   Color get tertiary3 => const Color(0xFF3AE3E0);
   Color get neutral => const Color(0xFF121212);
   Color get neutral2 => const Color(0xFF2F2F2F);
   Color get neutral3 => const Color(0xFF5D5D5D);
   Color get neutral4 => const Color(0xFF898989);
   Color get neutral5 => const Color(0xFFB7B7B7);
   Color get neutral6 => const Color(0xFFE5E5E5);
```

Comment:
We can also get rid of unwanted `neutral` color sets and use only default `neutral` color based on the requirement.

Reply:
generally yes, but many of them seem to be used in the ChatTheme and I wasn't really clear what I was touching or how to better infer the data, so I left this untouched for now. Ideally I want the entire chatTheme also to become a `chatThemeMaker` that given the `colorScheme` is just generated on the fly without hardcoded colors.